go.mod

```diff
@@ -79,4 +81,7 @@ replace (
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.0.0-20190918161442-d4c9c65c82af
 	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.0.0-20190918162410-e45c26d066f2
 	k8s.io/sample-controller => k8s.io/sample-controller v0.0.0-20190918161628-92eb3cb7496c
+
+	// TODO: Remove once https://github.com/kubernetes-sigs/controller-runtime/pull/588 or https://github.com/kubernetes-sigs/controller-runtime/pull/618 is merged.
+	sigs.k8s.io/controller-runtime => github.com/enxebre/controller-runtime v0.2.0-beta.1.0.20190930160522-58015f7fc885
```

Comment:
Let's hold until https://github.com/kubernetes-sigs/controller-runtime/pull/618 merges and we can use it.

/hold